app/main.py

```diff
@@ -4,3 +1,20 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+URL = "http://api.weatherapi.com/v1/current.json"
+city = "Paris"
+
+
+def get_weather() -> str:
+    key = os.getenv("API_KEY")
+    response = requests.get(
+        URL,
+        params={"q": city, "key": key},
+    )
+    return f"Temperature in Paris: {response.json()['current']['temp_c']}"
```

Comment:
The function `get_weather` does not have any input parameters. This makes it less reusable. Consider adding `city` and `key` as parameters to this function. This way, you can call it with different cities and keys without changing the code.